### PR TITLE
Ocrvs 6202

### PR DESCRIPTION
**Pull request by tahmidrahman-dsi**
https://github.com/opencrvs/opencrvs-farajaland/pull/805

### CODE REVIEW

File: packages/config/src/handlers/forms/validation.ts [hunk 1]
```diff
@@ -300,6 +312,9 @@ const OPTIONAL_FIELDS_IN_SECTION: Record<string, string[] | undefined> = {
     'iD',
```
Comment (Zangetsu101) [inline packages/config/src/handlers/forms/validation.ts:312]: @euanmillar Do we want to keep the 'iD' field here as an optional field for backwards compatibility or have the implementers change it to `{sectionName}NationalId` as a manual migration step

**Review by Zangetsu101 — APPROVED** [review]
looks good @tahmidrahman-dsi. Just that we need to update the 'iD' field check here: https://github.com/opencrvs/opencrvs-core/blob/develop/packages/client/src/components/Header/Header.tsx#L361 for showing/hiding the nationId search option. I wonder if there are other places like this

**Comment by euanmillar** [comment]
@tahmidrahman-dsi we have decided to put this into 1.3.3. Can you make a new PR based on the release-v1.3.3 branch and cherry-pick these commits onto it and re-submit this PR please?

**Comment by tahmidrahman-dsi** [comment]
https://github.com/opencrvs/opencrvs-farajaland/pull/805 The ID fields are no longer optional hence there is no need to validate those fields. Closing this PR

**Pull request closed without merging**